image/image.go

```diff
@@ -67,6 +67,7 @@ const (
 	IMAGE_TLV_ENC_EC256 = 0x32
 	IMAGE_TLV_AES_NONCE = 0x50
 	IMAGE_TLV_SECRET_ID = 0x60
+	IMAGE_TLV_SECTION = 0x70
```

Comment:
This should be aligned if `go fmt`  was used! :-P

Reply:
Btw, `0x50`, `0x60` and `0x70` are all downstream and not part of MCUboot. MCUboot already has its own `0x50` and `0x60` TLVs not related to these, so I just wonder when will the day come that you sync to MCUboot upstream and unexpected things happen! :-)

Reply:
yeah good points on the TLV numbers.  What should we do for proprietary TLV numbers?  I wonder if we shouldnt set aside some proprietary range.

Reply:
moved these to legacy numbers.  we'll keep them around until we update mcuboot locally and then remove them then.